server/src/models/database/recipe.ts

```diff
@@ -1,15 +1,24 @@
 import { IsNotEmpty, IsNumber, Length } from "class-validator";
+import { ValidationGroup } from "./../enums/validation-group";
 import { FoodItem, RecipeStep, SocoboUser } from "./../index";
 
 export class Recipe {
 
   public fields: Map<string, Function>;
 
   public id: number;
-  @IsNotEmpty()
-  @Length(1, 50)
+
+  @IsNotEmpty({
+    groups: [ ValidationGroup.RECIPE ]
```

Comment:
Yes. I think its possible to not use a group. But I need to check the documentation first.